README.md

```diff
@@ -44,7 +44,8 @@ pipeline:
     image: cschlosser/drone-ftps
     hostname: example.com:21
     secrets: [ ftp_username, ftp_password ]
-    secure: true
+    secure: true | false # true = use FTP(S), false = FTP without SSL
+    verify: true | false # true = strong SSL verification, false = supress SSL versification error
```

Comment:
typo: verification

Reply:
Please also add which is the default value

Reply:
the default value for `verify` ist `true`, like with `secure:`. 
do you mean add info about default value in README?

Reply:
Yes exactly. If we add this sort of documentation we should also make it clear what is the default value in the README :)